server/CMakeLists.txt

```diff
@@ -30,6 +30,17 @@ else()
 	endif()
 endif()
 
+if (AUDIOAPI STREQUAL jack)
+	# here we check for JACK metadata API
+	include(CheckIncludeFiles)
+	CHECK_INCLUDE_FILES("jack/metadata.h" JACK_USE_METADATA_API)
+	if(${JACK_USE_METADATA_API})
+		message(STATUS "using JACK metadata API")
+		add_definitions("-DSC_JACK_USE_METADATA_API")
```

Comment:
maybe use `target_compile_definitions` as target scoping is more robust than directory scoping